example/book.toml

```diff
@@ -3,7 +3,11 @@ title = "Example Book"
 
 [preprocessor.d2]
 command = "cargo run --manifest-path ../Cargo.toml"
+layout = "elk"
 inline = true
+# regular-font-path = "./fonts/Hack-Regular.ttf"
+# italic-font-path = "./fonts/Hack-Italic.ttf"
+# bold-font-path = "./fonts/Hack-Bold.ttf"
```

Comment:
curious to get your thoughts on this. The official docs say that all three should be supplied for consistency.

I wonder if there's a better way to expose these based on that.

possibly something like:

```toml
[preprocessor.d2]
command = "cargo run --manifest-path ../Cargo.toml"
layout = "elk"
inline = true
font-paths = { "regular = "./fonts/Hack-Regular.ttf", italic = "./fonts/Hack-Italic.ttf", bold = "./fonts/Hack-Bold.ttf" }
```

or equivalently,

```toml
[preprocessor.d2]
command = "cargo run --manifest-path ../Cargo.toml"
inline = true
layout = "elk"

[preprocessor.d2.font-paths]
bold = "./fonts/Hack-Bold.ttf"
italic = "./fonts/Hack-Italic.ttf"
regular = "./fonts/Hack-Regular.ttf"
```

that at least removes the redundancy of saying 'font-path' three times and better captures the intent (that all three are provided together)

thoughts?